app/src/app/[locale]/serverAction/page.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import { useFormState } from "react-dom";
+
+import {
+  Button,
+  Grid,
+  GridContainer,
+  Label,
+  TextInput,
+} from "@trussworks/react-uswds";
+
+import { updateServerData } from "../../serverActions/serverActionExample";
+
+const initialFormState = {
+  name: "",
+  email: "",
+};
+
+export default function SimpleForm() {
+  const [formData, updateFormData] = useFormState(
+    updateServerData,
+    initialFormState
+  );
+
+  const hasReturnedFormData = formData.name || formData.email;
+
+  return (
+    <GridContainer>
+      <Grid row>
+        <Grid col={12}>
+          <div className="usa-card">
+            <div className="usa-card__header">
+              <h2>Server Action Example</h2>
```

Comment:
nit: I think ideally the text in the example would use the internationalization library so we can ensure our example doesn't have any unexpected issues when utilizing it with this approach.

Reply:
Updated with i18n!